DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -127,7 +127,7 @@ const char* const TaggingVariableTokens[] = {
   /* [trackPtRatio]                             = */ "trackPtRatio",
   /* [trackPParRatio]                           = */ "trackPParRatio",
 
-  /* [trackIp2dVal]                             = */ "trackIp2dVal",
+  /* [trackIp2dSig]                             = */ "trackIp2dSig",
```

Comment:
do not use maps, do not use lists unless you have an algorithm that makes an optimal use of their peculiar data structures. They are very memory greedy and inefficient in particular for write once read many cases. 
if possible use a (sorted) std::vector.   unordered_map is also a possibility